lib/api-export.js

```diff
@@ -22,32 +18,24 @@ class ApiExport extends ApiSaveData {
 	async validate() {
 
 		await super.validate();
-
-		const exportValidator = new ExportValidator(this);
-
-		exportValidator.validateData();
-		exportValidator.validateModel(this.dataToSave.main.entity);
-		exportValidator.validateController(this.dataToSave.main.entity);
+		this.exportValidator = this.session.getSessionInstance(ExportValidator);
+		this.dataToSave.main = this.exportValidator.validate(this.dataToSave.main);
 	}
 
-	async format(document) {
-
-		const userEmail = await this.getUserEmail(this.session.userId);
-
-		if(!userEmail)
-			throw new Error('Invalid User');
-
-		return exportFormatter(document, this.session.userId, userEmail, this.statuses.pending);
-	}
+	async format(exportDocument) {
 
-	async getUserEmail(id) {
-		const response = await this.session.getSessionInstance(MsCall).call('id', 'user', 'get', null, null, { id });
+		const userEmail = await this.exportValidator.validateEmail();
 
-		return response.body && response.body.email;
+		return {
+			...exportDocument,
+			userEmail,
+			userCreated: this.session.userId,
+			status: this.statuses.created
+		};
 	}
 
-	postSaveHook(id) {
-		return createdEvent(id, this.session.clientCode);
+	postSaveHook(id, dataSaved) {
+		return Invoker.clientCall('ExportProcess', this.session.clientCode, { id, ...dataSaved });
```

Comment:
Aunque yo sé que es mas comodo usarlo asi, vamos a usar como un standard que el payload de los invoke no tengan un objeto tirado asi, por eso en el ticket te lo piden asi.

```suggestion
		return Invoker.clientCall('ExportProcess', this.session.clientCode, { exportDocument: { id, ...dataSaved } });
```